### PR TITLE
校对 6.3

### PR DESCRIPTION
**Pull request by ykla**

### CODE REVIEW

File: di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
 # 6.3.生产力工具
 
-当提到生产力工具时，用户往往倾向于使用办公套件或者易于使用的文字处理软件。虽然有些桌面环境比如 KDE，他会提供一个默认的办公套件，但是 FreeBSD 并没有提供默认的生产力工具包，你需要自己安装想要的办公套件和其他图形化的文字处理程序。FreeBSD 提供的生产力工具与你安装的窗口管理器或桌面的类型无关。
+当提到生产力工具时，用户往往倾向于使用办公套件或者易于使用的文字处理软件。虽然有些[桌面环境](https://docs.freebsd.org/en/books/handbook/book/#x11-wm)比如 KDE，他会提供一个默认的办公套件，但是 FreeBSD 并没有提供默认的生产力工具包，你需要自己安装想要的办公套件和其他图形化的文字处理程序。FreeBSD 提供的生产力工具与你安装的窗口管理器或桌面的类型无关。
```
Comment (rowheel) [inline di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md:3]:
```suggestion
当提到生产力工具时，用户往往倾向于使用办公套件或者易于使用的文字处理软件。虽然有些[桌面环境](https://docs.freebsd.org/en/books/handbook/book/#x11-wm)（比如 KDE）会提供一个默认的办公套件，但是 FreeBSD 并没有提供默认的生产力工具包，你需要自己安装想要的办公套件和其他图形化的文字处理程序。FreeBSD 提供的生产力工具与你安装的窗口管理器或桌面的类型无关。
```

> 但是 FreeBSD 并没有提供默认的生产力工具包

Besides, I don't quite get the logic behind the but-phrase here. Does it mean KDE provides an office suite, but that suite is not provided by FreeBSD, it's provided by KDE?

File: di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md [hunk 2]
```diff
@@ -65,15 +65,15 @@ AbiWord 可以导入或导出许多文件格式，包括一些专有格式，如
 # make install clean
 ```
 
-ports 的 `图像类别` 中包含了许多跟 GIMP 有关的插件、帮助文档和用户手册。
+ports 的图像类别(<https://www.freebsd.org/ports/graphics/>)中包含了许多跟 GIMP 有关的插件、帮助文档和用户手册。
```
Comment (rowheel) [inline di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md:68]:
```suggestion
ports 的 (graphics 类别)[https://www.freebsd.org/ports/graphics/] 中包含了许多跟 GIMP 有关的插件、帮助文档和用户手册。
```

File: di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md [hunk 3]
```diff
@@ -65,15 +65,15 @@ AbiWord 可以导入或导出许多文件格式，包括一些专有格式，如
...
 
 ## 6.3.4. Apache OpenOffice
 
 Apache OpenOffice 是一个开源的办公套件，在 Apache 软件基金会的孵化器的支持下开发。它包括一个完整的办公生产力套件应包含的所有应用程序：一个文字处理器、电子表格处理器、幻灯片管理器和绘图程序。它的用户界面与其他办公套件相似，并且可以导入和导出各种流行的文件格式。它有许多不同的语言版本，并且支持界面、拼写检查器和字典等的本地化。
 
-Apache OpenOffice 的文字处理器使用 XML 文件格式来增加可移植性和灵活性。其电子表格程序具有一个宏处理程序，可以与外部数据库连接。Apache OpenOffice 可以在 Windows® , Solaris™ , Linux®, FreeBSD , 和 Mac OS® X 上运行。关于 Apache OpenOffice 的更多信息，可以参考 openoffice.org。关于 FreeBSD 的具体信息，请参考 porting.openoffice.org/freebsd/。
+Apache OpenOffice 的文字处理器使用 XML 文件格式来增加可移植性和灵活性。其电子表格程序具有一个宏处理程序，可以与外部数据库连接。Apache OpenOffice 可以在 Windows® , Solaris™ , Linux®, FreeBSD , 和 Mac OS® X 上运行。关于 Apache OpenOffice 的更多信息，可以参考 [openoffice.org](http://openoffice.org/)。关于 FreeBSD 的具体信息，请参考 [porting.openoffice.org/freebsd/](http://porting.openoffice.org/freebsd/)。
 
-欲通过 package 安装 Apache OpenOffice，请使用：
+要通过软件安装 Apache OpenOffice，请使用：
```
Comment (rowheel) [inline di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md:76]:
```suggestion
要通过 pkg 工具安装 Apache OpenOffice，请使用：
```

File: di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md [hunk 4]
```diff
@@ -96,28 +96,28 @@ Apache OpenOffice 的文字处理器使用 XML 文件格式来增加可移植性
...
 
-欲安装英文版的 LibreOffice，请执行：
+要安装英文版的 LibreOffice：
 
 ```
 # pkg install libreoffice
 ```
 
-在 ports 的编辑器类别 (freebsd.org/ports/editors/) 中包含了许多 LibreOffice 的本地化版本，若你打算安装一个本地化的版本，请将上述命令中的 `libreoffice` 替换为本地化版本软件包的具体名称。
+在 ports 的编辑器类别 ([freebsd.org/ports/editors/](https://www.freebsd.org/ports/editors/)) 中包含了许多 LibreOffice 的本地化版本，若你打算安装一个本地化的版本，请将上述命令中的 `libreoffice` 替换为本地化版本软件包的具体名称。
```
Comment (rowheel) [inline di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md:120]:
```suggestion
在 ports 的 (editors 类别)[https://www.freebsd.org/ports/editors/] 中包含了许多 LibreOffice 的本地化版本，若你打算安装一个本地化的版本，请将上述命令中的 `libreoffice` 替换为本地化版本软件包的具体名称。
```

File: di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md [hunk 5]
```diff
@@ -136,4 +136,4 @@ LibreOffice 的文字处理器使用 XML 文件格式来增加可移植性和灵
 
 > **注意**
 >
-> 欲构建本地化版本的 LibreOffice，请先使用 `cd` 进入你想要的语言版本所对应的 port 目录。你可以在 Port 的 `编辑器类别`(freebsd.org/ports/editors/)中找到支持的语言。
+> 要构建本地化版本的 LibreOffice，请先使用 `cd` 进入你想要的语言版本所对应的 port 目录。你可以在 ports 的编辑器类别([freebsd.org/ports/editors/](https://www.freebsd.org/ports/editors/))中找到受支持的语言。
```
Comment (rowheel) [inline di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md:139]:
```suggestion
> 要构建本地化版本的 LibreOffice，请先使用 `cd` 进入你想要的语言版本所对应的 port 目录。你可以在 ports 的 (editors 类别)[https://www.freebsd.org/ports/editors/] 中找到受支持的语言。
```

**Review by rowheel — CHANGES_REQUESTED** [review]

File: di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md (hunk 1 above)
Comment (ykla) [reply di-6-zhang-zhuo-mian-ying-yong-cheng-xu/6.3.-sheng-chan-li-gong-ju.md:3]: FreeBSD 不自带任何 GUI 软件的。

**Pull request merged**